src/geode/model/helpers/component_mesh_queries.cpp

```diff
@@ -233,47 +200,55 @@ namespace
         geode::detail::OrientedVertexCycle< geode::PolygonVertices >
             polygon_unique_vertices_cycle{ polygon_unique_vertices };
         if( polygon_unique_vertices_cycle.is_opposite(
-                { facets_unique_vertices[0] } ) )
+                { facets_unique_vertices[0].vertices } ) )
         {
             return { facets_block_vertices, facets_unique_vertices, false };
```

Comment:
Will the compiler be able to move facets_block_vertices from output to construct the object in place here?